server/pulp/server/async/scheduler.py

```diff
@@ -178,6 +178,9 @@ def monitor_events(self):
                 'task-succeeded': self._failure_watcher.handle_succeeded_task,
                 'worker-heartbeat': worker_watcher.handle_worker_heartbeat,
                 'worker-offline': worker_watcher.handle_worker_offline,
+                # Use the heartbeat handler for the worker-online event as this will handle
+                # worker discovery if needed
```

Comment:
This comment does not seem to contribute anything beyond what I get from just reading one line of code.  At the risk of heresy,  consider removing the comment to improve code readability.  Just my $0.02.